src/main/webapp/app/exercises/programming/manage/grading/programming-exercise-configure-grading.component.html

```diff
@@ -98,7 +98,9 @@ <h2 class="mb-1 fw-medium" jhiTranslate="artemisApp.programmingExercise.configur
                             <ngx-datatable-column name="Test Name" prop="testName">
                                 <ng-template ngx-datatable-header-template let-sortFn="sortFn">
                                     <span class="datatable-header-cell-wrapper" (click)="sortFn()">
-                                        <span class="datatable-header-cell-label bold sortable">Test Name</span>
+                                        <span class="datatable-header-cell-label bold sortable">{{
+                                            'artemisApp.programmingExercise.configureGrading.gradingTableHeader.testName' | artemisTranslate
+                                        }}</span>
```

Comment:
```suggestion
                                        <span class="datatable-header-cell-label bold sortable">
                                            {{'artemisApp.programmingExercise.configureGrading.gradingTableHeader.testName' | artemisTranslate}}
                                        </span>
```
This style would make more sense to me. Unless eslint complains. If you agree, please replace the other occurrences in your PR.